include/h2o/url.h

```diff
@@ -121,4 +121,27 @@ inline h2o_iovec_t h2o_url_stringify(h2o_mem_pool_t *pool, const h2o_url_t *url)
     return h2o_url_resolve(pool, url, NULL, &tmp);
 }
 
+static inline int h2o_url_host_is_unix_path(h2o_iovec_t host)
+{
+    if (host.len < 6) {
+        return 0;
+    }
+    return h2o_memis(host.base, 6, H2O_STRLIT("unix:/"));
```

Comment:
Should we better compare against `unix:` (without slash)? IIRC we do not have a restriction that all Unix sockets are absolute.